modules/fundamental/src/package/model/details/qualified_package.rs

```diff
@@ -44,6 +54,36 @@ impl QualifiedPackageDetails {
                 .ok_or(Error::Data("underlying package missing".to_string()))?
         };
 
+        /*
```

Comment:
I would prefer not to add more commented out code.

Reply:
Right on.